learn/advanced/updating.md

```diff
@@ -1,27 +1,25 @@
 # Update to the latest Meilisearch version
 
-Currently, Meilisearch databases can only be opened by the Meilisearch version you used to create them. The following guide will walk you through all the steps to migrate an existing database from an older version of Meilisearch to the most recent one.
-
-If you already have a Meilisearch database with some data you don’t want to lose, you are in the right place!
-
 ::: danger
 This guide does not work for versions below v0.15. For more information, [contact support](https://discord.gg/meilisearch).
 :::
 
-## Step 1: Verify your database version
+Currently, Meilisearch databases can only be safely opened by the Meilisearch version you used to create them. The following guide will walk you through all the steps to migrate an existing database from an older version of Meilisearch to the most recent one.
 
-Before we begin, you need to verify the version of Meilisearch that's compatible with your database, in other words, the version that indexed the data. You can do so by launching a Meilisearch instance:
+If you're updating your Meilisearch instance on cloud platforms like DigitalOcean, AWS, or GCP, ensure that you can connect to your cloud instance via SSH. Depending on the user you are connecting with (root, admin, etc.), you may need to prefix some commands with `sudo`.
 
-```bash
-./meilisearch --master-key="MASTER_KEY"
-```
+::: tip
+If you are using v0.22 or above, use our [migration script](https://github.com/meilisearch/meilisearch-migration).
+:::
 
-If Meilisearch launches successfully, use the get version endpoint, note your `pkgVersion`, and [proceed to the next step](#step-2-set-all-fields-as-displayed-attributes).
+## Step 1: Verify your database version
+
+Before we begin, you need to verify the version of Meilisearch that's compatible with your database. Use the get version endpoint and note your `pkgVersion`:
 
 <CodeSamples id="updating_guide_check_version_new_authorization_header" />
 
-::: note
-If Meilisearch returns a [`missing_authorization_header`](/reference/errors/error_codes.md#missing-authorization-header) error code, you might be using v0.24 or below. Change the authorization header to `X-MEILI-API-KEY: apiKey`:
+::: warning
```

Comment:
As mentioned in a separate comment, I recommend generalizing this warning and moving it to the beginning of the guide, so it doesn't break up the reader's flow and make the process seem more complicated than it is.